### PR TITLE
Added ability to control social sharing for individual sites

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6578

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: docs/credentials_admin.rst [hunk 1]
```diff
@@ -25,92 +25,21 @@ Each site can have only one configuration.
 
 
 Add Another Site and Site Configuration
------------------------------------------
-
-If you want to add more sites, partners, and site configurations, you can use
-the ``create_or_update_site`` command. The following options are available for
```
Comment (rlucioni) [inline docs/credentials_admin.rst]: Big 👍 for this.

File: docs/credentials_admin.rst [hunk 2]
```diff
@@ -198,4 +127,4 @@ The following image shows the ``ProgramCertificate`` model in the Django adminis
     :width: 600px
     :alt: Populated program certificate model
 
-In ``ProgramCertificate`` if ``Use organization name`` is checked then organization full name will be displayed on the certificate else short name would be displayed.
+If `Use organization name` is checked then organization full name will be displayed on the certificate else short name would be displayed.
```
Comment (rlucioni) [inline docs/credentials_admin.rst]: s/name would be/name will be

Comment (clintonb) [inline docs/credentials_admin.rst]: This will be fixed in a subsequent PR...in about 10 minutes.

**Comment by clintonb** [comment]
Merged with #192.

**Pull request closed without merging**